### PR TITLE
Reordered ModelType enum

### LINKED ISSUE #37: [cleanup] match ModelType orderering in formatting rules
Please use consistent ModelType ordering in (db|message)FormattingRules(_spec).js.

Alternatively it might be more natural to just change order of ModelType enum itself.

### PR DESCRIPTION
**Pull request by martinayora**
Fixes #37

### CODE REVIEW

File: rest/src/db/dbFormattingRules.js [hunk 1]
```diff
@@ -29,9 +29,9 @@ module.exports = {
 	[ModelType.none]: value => value,
```
Comment (Vektrat) [inline rest/src/db/dbFormattingRules.js]: should we alphabetize `none` or is it considered special?

**Comment by Vektrat** [comment]
maybe should order `SchemaType.js` as well?

File: rest/test/db/dbFormattingRules_spec.js [hunk 2]
```diff
@@ -21,8 +21,8 @@
 const catapult = require('catapult-sdk');
```
Comment (Jaguar0625) [inline rest/test/db/dbFormattingRules_spec.js]: seem to be missing string test

File: rest/test/server/messageFormattingRules_spec.js [hunk 3]
```diff
@@ -48,52 +48,52 @@ describe('message formatting rules', () => {
...
-		const result = formattingRules[ModelType.string](object);
+		const result = formattingRules[ModelType.uint64](object);
 
 		// Assert:
-		expect(result).to.equal('catapult');
+		expect(result).to.deep.equal([1, 2]);
 	});
 
-	it('can format status code type', () => {
+	it('can format uint16 type', () => {
```
Comment (Jaguar0625) [inline rest/test/server/messageFormattingRules_spec.js]: 64 + 16 should be swapped

**Pull request merged**